code/game/jobs/job/security.dm

```diff
@@ -182,8 +182,7 @@
 				H.equip_or_collect(new /obj/item/clothing/suit/storage/det_suit(H), slot_wear_suit)
 				H.equip_or_collect(new /obj/item/clothing/head/det_hat(H), slot_head)
 				H.equip_or_collect(new /obj/item/clothing/shoes/brown(H), slot_shoes)
-	H.equip_or_collect(new /obj/item/weapon/lighter/zippo(H), slot_l_store)
-
+		H.equip_or_collect(new /obj/item/weapon/lighter/zippo(H), slot_l_store)
```

Comment:
1. Don't put this here, you've put it inside the if(alt title) check which while it should always be true, it's a weird place for it, put it next to the black gloves above
2. This isn't atomic. Probably no one will object but when we talk about atomic PRs we're saying "it does one thing" so that it's more clear what people are voting about. This might be a problem for example if someone wanted to vote FOR the return of zippo but against the new title, or something like that.
3. This is a substance change and should be put in the changelog.